ui/src/app/metadata/detail/metadata-manage/metadata-manage.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, Input } from "@angular/core";
+import { Title } from "@angular/platform-browser";
+import { ActivatedRoute, Router } from "@angular/router";
+
+import { AbstractDataService } from "src/app/data/abstract-data.service";
+import { RichSkillService } from "src/app/richskill/service/rich-skill.service";
+import { ToastService } from "src/app/toast/toast.service";
+import { AbstractMetadataDetailComponent } from "../abstract-metadata-detail.component";
+
+
+@Component({
+  selector: "app-metadata-manage",
+  templateUrl: "./metadata-manage.component.html"
+})
+export class MetadataManageComponent extends AbstractMetadataDetailComponent {
+
+  constructor(
+    protected router: Router,
+    protected route: ActivatedRoute,
+    protected metadataService: AbstractDataService,
```

Comment:
This error `constructor was not compatible with Dependency Injection.` appears in the console, I think is related with this service.